dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GitHubUtil.java

```diff
@@ -169,6 +169,7 @@ public GHContent tryRetrievingContent(GHRepository repo, String path, String bra
                 String forkName = s.substring(s.lastIndexOf('/') + 1);
                 log.info(forkName);
                 if (!repoNamesSet.contains(forkName)) {
+                    log.info("Forking is still in progress for {}" , forkName);
```

Comment:
Will this get chatty in the logs? Would it maybe be better for `debug` mode? A screenshot of output in the PR comment might be helpful to see what it looks like.

Reply:
This won't chatty in the logs because we have 60 sec sleep. This log is important for us to know if the execution has gone into infinite loop again.
<img width="1298" alt="Screen Shot 2019-05-06 at 7 49 16 PM" src="https://user-images.githubusercontent.com/1262405/57268294-17b92580-7038-11e9-80fc-409344636c1c.png">

Reply:
Updated PR.

Reply:
Does a user care about this especially since there are already a lot of log messages? I was wondering if you could send a screenshot of a bunch of log messages to see how this looks in the flow.